ts/defs.bzl

```diff
@@ -439,6 +494,7 @@ def ts_project(
         tsc = tsc,
         tsc_worker = tsc_worker,
         transpile = -1 if not transpiler else int(transpiler == "tsc"),
+        declaration_transpile = declaration_transpiler != None,
```

Comment:
should be `declarations_target_name != None` since there are cases where `declaration_transpiler` is ignored

Reply:
Actually I think I'm going to leave this (and the above line) as-is.

These variables just tell the rule if alternate transpilers exist, not if the current tsconfig will vs will not use them.